### ISSUE TITLE
[Q] Keil or Code Composer ?

### ISSUE THREAD

**Issue by jakubczaplicki** [fanweng/Realtime-Bluetooth-Networks-Lab3-Time-Management#1]
Hi Fan, I don't think there's a way to send a private message on github, so I am submitting this issue instead. I see you're actively working on this MOOC. Are you using Keil or Code Composer as the IDE ?

**Comment by fanweng** [comment]
Hi Jakub, I am using Keil uVision v5.2 as the IDE. You can find the link and installation instructions in the README of my [Realtime-Bluetooth-Networks-Lab1-Run-IO](https://github.com/fanweng/Realtime-Bluetooth-Networks-Lab1-Run-IO/blob/master/README.md) repository.

**Comment by jakubczaplicki** [comment]
Yup. Thanks. I do have Keil set up. I now moved from Windows to Mac. I got fed up with switching between two laptops, and I am in the process of importing the code do Code Composer on Mac. So far I managed to run the BoardSupportPackage_MSP432, but not the Labs. I am looking for someone who could help me a bit ;)

![ti_example](https://user-images.githubusercontent.com/147901/38754245-4837bad6-3f61-11e8-9675-29331d8c3e0b.jpeg)

**Comment by fanweng** [comment]
Which lab are you doing right now? I am studying the course materials for Lab3 and hopefully will get it done in next week. We can keep in touch and have discussions.

**Comment by jakubczaplicki** [comment]
I am going to start working on Lab2 soon. So far I got my finished Lab 1 to compile under Code Composer. I had to remove grader and profiler though. I don't think there's a Mac version of TExaSdisplay, so I won't be able to confirm if my solution is indeed the correct one. Oh well..

**Comment by fanweng** [comment]
Unfortunately, my device is TM4C123. Otherwise, I can clone your lab repository and test it for you. But here are two possible ways you can do to get task execution profiles: 1) get a real logic analyzer; 2) toggle GPIO pins for each individual task and use a oscilloscope to monitor those GPIOs. An oscilloscope is much cheaper, by the way :)

**Comment by jakubczaplicki** [comment]
hi Fan,
I'd appreciate if you could have a quick look at my lab2: https://github.com/jakubczaplicki/rtbn-msp432/tree/lab2/Lab2_MSP432  . It compiles, and deploys the code OK, but the screen doesn't show any data. In fact I think the code exists pretty quickly after startup. btw.I am waiting for a simple logic analyzer already.

**Comment by fanweng** [comment]
I checked your repository and found you did not finish the SysTick_Handler and StartOS functions in osasm.asm.

**Comment by jakubczaplicki** [comment]
True. Thanks!
